elasticdl/client/client.py

```diff
@@ -31,7 +33,9 @@ def _build_docker_image(m_path, m_file, m_file_in_docker, timestamp):
 
     with open(new_dfile, 'a') as df:
         df.write("COPY " + m_file + " " + m_file_in_docker)
```

Comment:
I've made a PR with the fix. It might be the best way to handle it, due to the that it will directly be forwarded to docker compose.
https://github.com/docker/docker-py/pull/2456